src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user;
+    private static RegistrationService registrationService;
+    private static final long ID = 0;
+    private static final String LOGIN_VALID = "aboba@gmail.com";
+    private static final String PASSWORD_VALID = "123456";
+    private static final String PASSWORD_AND_EMAIL_INVALID = "12345";
```

Comment:
In my opinion it's better to create invalid field for email and password separately